scripts/aptly/aptly_importer.py

```diff
@@ -321,11 +338,19 @@ def run(self):
 
 def main():
     """
-    Usage: python3 aptly_importer.py [--simulate-repo-import] <config_file>
+    Usage: python3 aptly_importer.py [parameters] <config_file>
```

Comment:
Does it make sense to preserve the explicit parameters in the usage text for quick reference?